tmk/cpp/Makefile

```diff
@@ -1,7 +1,7 @@
 # ----------------------------------------------------------------
 CXX=g++ # please customize according to your system --- Note: Apple clang is not supported 
 CC=$(CXX) -O2 -std=c++14 -I../.. #
-FFMPEG=/Users/barrett/homebrew/bin/ffmpeg # please customize according to your installation
+FFMPEG=/usr/bin/ffmpeg # please customize according to your installation (but do not commit)
```

Comment:
non-blocking: Should we make this an environment variable?

Reply:
yes, good follow up, I don't know why it wasn't one from the start.